src/components/thumbnail-maker/TagSheet.tsx

```
@@ -52,30 +57,38 @@ function TagSheet({
                     className="h-fit w-fit origin-top-left scale-50 transform transition-all duration-300 ease-in-out"
-                    onClick={() => setTag(currentTag)}
+                    onClick={() =>
+                      setTag({
+                        ...currentTag,
+                        text: tag.text,
+                      })
+                    }
                   >
-                    <TagItemView tag={currentTag} />
+                    <TagItemView
+                      tag={currentTag}
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)